test/cim/cim_creators.py

```diff
@@ -427,7 +427,7 @@ def create_connectivity_node_container(include_runtime: bool):
 
 def create_equipment(include_runtime: bool):
     runtime = {
-        "current_feeders": lists(builds(Feeder, **create_identified_object(include_runtime)), min_size=1, max_size=2)
+        "current_containers": lists(builds(Feeder, **create_identified_object(include_runtime)), min_size=1, max_size=2)
```

Comment:
todo: This is only creating a collection of `Feeder` which will hide any left over issues with the change to supporting other equipment containers. It should be updated to use something like `equipment_containers` using `sampled_equipment_container` , but only selecting valid current containers.